support-frontend/yarn.lock

```diff
@@ -80,6 +80,15 @@
     jsesc "^2.5.1"
     source-map "^0.5.0"
 
+"@babel/generator@^7.18.2":
```

Comment:
I'm not sure why there's so many changes to the lockfile as you don't seem to have added any new dependencies. Probably best to revert this file?